### PR TITLE
Changed the text "Click here" to "Double-click here" in the editor placeholder (for #19)

### PR DESCRIPTION
**Pull request by PatrykSzwer**

### CODE REVIEW

**Comment by tomalec** [comment]
Now, it feels like Christmass https://www.boredpanda.com/my-friends-sister-had-some-fun-with-his-gift-wrapping-2/ ;)

**Comment by tomalec** [comment]
https://github.com/Starcounter/Blending/issues/21#issuecomment-356000686

File: starcounter-layout-html-editor.html [hunk 1]
```diff
@@ -97,7 +97,7 @@
             <input
                 list="selector-autocomplete"
                 type="text"
-                placeholder="Click here to pick the part of a screen that you want to edit"
+                placeholder="Double-click here to pick the part of a screen that you want to edit"
```
Comment (warpech) [inline starcounter-layout-html-editor.html]: Given the @tomalec's comment, maybe do it like this: `Type a selector of <starcounter-include> or double-click to choose`

Comment (PatrykSzwer) [reply starcounter-layout-html-editor.html]: Okey dokie, done!

**Pull request merged**